blog/2018-07-17-UnitTestsInXslxFileForConferenceScheduling.adoc

```diff
@@ -0,0 +1,122 @@
+= Unit tests in .xlsx files
+:page-interpolate: true
+:awestruct-author: MusaTalluzi
+:awestruct-layout: blogPostBase
+:awestruct-tags: [useCase]
+
+After you implement a planning problem, you must write unit tests for every score rule.
+
+
+== Traditional unit tests
+
+In Conference Scheduling example, if you want to test room conflict constraint (hard penalty per pair of talks in the same room in overlapping timeslots)
```

Comment:
Try to favor using active voice, see https://developers.google.com/style/voice

So instead of "... if you want to test room ..." simply "... to test room ...".
Instead of you "... you might write something like this:", simply "... write:"

- Shorter sentences that have the same information are better. (less is more)
- Show confidence and authority. Otherwise you're sending subliminal messages to the reader that he/she shouldn't waste his/her time reading this. Do be humble when it comes down to claiming things like "the best" etc without scientific proof.